.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  schedule:
+    # Use <https://crontab.guru> to conveniently edit cron schedule.
+    - cron: "0 7 * * *" # At 07:00 UTC every day.
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    env:
+      RUST_BACKTRACE: 1
+    strategy:
+      matrix:
+        build: [linux64, macos, win32]
+        include:
+          - build: linux64
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
```

Comment:
Since we do cross-testing, it might be better to call this `host_target`.

Reply:
for win32, it isn't host_target, because the builder is x86_64.
Don't worry about this, since I will use OS name instead of target triple for cross-testing.

Reply:
> it isn't host_target, because the builder is x86_64.

It should be the host target though! By "host target" I mean the target of the host rustc that gets installed. Even if this is an x86_64 builder, we want to install the 32bit rustc. So, the host-target should be a 32bit target still.

Reply:
if you say so.